### PR TITLE
FOGL-3397 Added mandatory in asset name config

### PR DESCRIPTION
**Pull request by rajeshkumar426**
Signed-off-by: rajeshkumar426 <rajeshkumar426@outlook.com>

### CODE REVIEW

**Review by MarkRiddoch — APPROVED** [review]

**Comment by ashish-jabble** [comment]
@rajeshkumar426 Please resolve conflicts.

File: plugin.cpp [hunk 1]
```diff
@@ -21,6 +21,14 @@
...
 
 #define PLUGIN_NAME "Csv"
+#define CONFIG	"{\"plugin\" : { \"description\" : \"" PLUGIN_NAME " C south plugin\", " \
+			"\"type\" : \"string\", \"default\" : \"" PLUGIN_NAME "\", \"readonly\" : \"true\" }, " \
+		"\"asset\" : { \"description\" : \"Asset name\", " \
+			"\"type\" : \"string\", \"default\" : \"Vibration\", \"order\": \"1\", \"displayName\": \"Asset Name\", \"mandatory\": \"true\" }," \
+		"\"file\" : { \"description\" : \"Name of the file to read\", " \
+			"\"type\" : \"string\", \"default\" : \"\", \"order\": \"2\", \"displayName\": \"Path Of File\" }" \
+		"} "
+
```
Comment (ashish-jabble) [inline plugin.cpp:24]: No need of this define `CONFIG`, as we moved this to `default_config` as with latest develop.

**Review by ashish-jabble — CHANGES_REQUESTED** [review]
fix wrong conflict resolution

**Review by ashish-jabble — APPROVED** [review]
Please do squash & merge

**Pull request merged**